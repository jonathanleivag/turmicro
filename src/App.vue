<template>
  <h1>hola mundo 1</h1>
  <h2>hola mundo 2</h2>
  <h3>hola mundo 3</h3>
  <h4>hola mundo 4</h4>
  <h5>hola mundo 5</h5>
  <h6>hola mundo 6</h6>
</template>
