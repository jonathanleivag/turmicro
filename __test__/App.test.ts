import { mount } from '@vue/test-utils'

import App from '../src/App.vue'

describe('Component app', () => {
  test('Snapshot de app', () => {
    expect(App).toBeTruthy()
    const wrapper = mount(App)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
