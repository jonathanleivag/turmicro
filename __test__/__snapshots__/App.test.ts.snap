// Vitest Snapshot v1

exports[`Component app > Snapshot de app 1`] = `
"<header class=\\"hero\\">
  <div class=\\"horo__logo\\">
    <h3>TurMicro</h3>
  </div>
</header>
<main class=\\"main\\">
  <!--v-if-->
  <div class=\\"card\\">
    <div class=\\"card__header\\">
      <h3 class=\\"card__title\\">Puntos acomulados</h3>
    </div>
    <div class=\\"card__body\\"></div>
  </div>
  <!--v-if-->
  <!--v-if-->
</main>"
`;
